apps/docs/content/guides/local-development/declarative-database-schemas.mdx

```diff
@@ -216,6 +216,16 @@ Your schema files are run in lexicographic order. If you need to declare foreign
         └── 20241006112233_add_managers_table.sql
 ```
 
+To specify a custom order for applying the schemas, you can declare them explicitly in `config.toml`. Any glob patterns will evaluated, deduplicated, and sorted in lexicographic order. For example, the following pattern ensures `employees.sql` is always executed first.
```

Comment:
For small projects with only a few tables, the default schema order may be sufficient. However, as your project grows, you might need more control over the order in which schemas are applied.